synfig-studio/src/synfigapp/actions/valuedesccreatechildbone.cpp

```diff
@@ -216,7 +215,9 @@ Action::ValueDescCreateChildBone::prepare()
 
 	Action::Handle action;
 
-	action = ValueNodeStaticListInsert::create();
+	action = Action::create("ValueNodeStaticListInsert");
```

Comment:
Is there any reason to use this approach?

Reply:
Just for avoiding `#include "valuenodestaticlistinsert.h"`

Reply:
Ok